api/README.md

```diff
@@ -21,14 +21,56 @@ __Parameters__
 
 __`sha`__ : SHA[0:10] of the runs to get, or the keyword `latest`. Defaults to `latest`.
 
+__`product`__ : Product to include (repeated param), e.g. `chrome` or `firefox-60`
+
 __`from`__ : RFC3339 timestamp, for which to include runs that occured after the given time.
 
 __`max-count`__ : Maximum number of runs to get (for each browser). Only relevant when `sha` is `latest`. Maximum of 500.
 
+#### Examples
+
+- https://wpt.fyi/api/runs?product=chrome&product=safari
+- https://wpt.fyi/api/runs?product=chrome&from=2018-01-01T00:00:00Z&max-count=10
+
+__Example JSON__
+
+    [
+      {
+        "browser_name": "chrome",
+        "browser_version": "67.0.3396.62",
+        "os_name": "linux",
+        "os_version": "4.4",
+        "revision": "2bd11b91d4",
+        "full_revision_hash": "2bd11b91d490ddd5237bcb6d8149a7f25faaa101",
+        "results_url": "https://storage.googleapis.com/wptd/2bd11b91d4/chrome-stable-linux-summary.json.gz",
+        "created_at": "2018-06-05T08:27:30.627865Z",
+        "raw_results_url": "https://storage.googleapis.com/wptd-results/2bd11b91d490ddd5237bcb6d8149a7f25faaa101/chrome_67.0.3396.62_linux_4.4/report.json"
+      }
+    ]
+
 ### /api/runs/{id}
 
 Gets a specific (single) TestRun metadata by its datastore ID.
 
+#### Example
+
+https://wpt.fyi/api/runs/5164888561287168
+
+__Example JSON__
+
+    {
+      "id": "5164888561287168",
+      "browser_name": "chrome",
+      "browser_version": "67.0.3396.62",
+      "os_name": "linux",
+      "os_version": "4.4",
+      "revision": "2bd11b91d4",
+      "full_revision_hash": "2bd11b91d490ddd5237bcb6d8149a7f25faaa101",
+      "results_url": "https://storage.googleapis.com/wptd/2bd11b91d4/chrome-stable-linux-summary.json.gz",
+      "created_at": "2018-06-05T08:27:30.627865Z",
+      "raw_results_url": "https://storage.googleapis.com/wptd-results/2bd11b91d490ddd5237bcb6d8149a7f25faaa101/chrome_67.0.3396.62_linux_4.4/report.json"
+    }
+
```

Comment:
What *is* a "run"? What does this data represent?

Reply:
Added an overview, see `#test-run-entities`